Lombiq.Tests.UI/Services/OrchardCoreUITestExecutorConfiguration.cs

```diff
@@ -25,10 +25,36 @@ public class OrchardCoreUITestExecutorConfiguration
     public OrchardCoreConfiguration OrchardCoreConfiguration { get; set; }
 
     public int MaxRetryCount { get; set; } =
-        TestConfigurationManager.GetIntConfiguration("OrchardCoreUITestExecutorConfiguration:MaxRetryCount", 2);
+        TestConfigurationManager.GetIntConfiguration(
+            $"{nameof(OrchardCoreUITestExecutorConfiguration)}:{nameof(MaxRetryCount)}",
+            2);
 
     public TimeSpan RetryInterval { get; set; } =
-        TimeSpan.FromSeconds(TestConfigurationManager.GetIntConfiguration("OrchardCoreUITestExecutorConfiguration:RetryIntervalSeconds", 0));
+        TimeSpan.FromSeconds(TestConfigurationManager.GetIntConfiguration(
+            $"{nameof(OrchardCoreUITestExecutorConfiguration)}:RetryIntervalSeconds",
+            0));
+
+    /// <summary>
+    /// Gets or sets how many tests should run at the same time. Use a value of 0 to indicate that you would like the
+    /// default behavior. Use a value of -1 to indicate that you do not wish to limit the number of tests running at the
+    /// same time. The default behavior and 0 uses the <see cref="Environment.ProcessorCount"/> property. Set any other
+    /// positive integer to limit to the exact number.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// The XUnit MaxParrallelThreads property controls only the threads, not the actual processes started. See <see
```

Comment:
Also fix the same typo in `MaxParrallelTests`.

```suggestion
    /// The XUnit MaxParallelThreads property controls only the threads, not the actual processes started. See <see
```